### PR TITLE
Stardew Valley emoting system

### PR DESCRIPTION
**Pull request by fqqf**
Stardew emotes system
image
image
image
image

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

:cl: Miracula
- add: Added cloud emotes

### CODE REVIEW

**Comment by fqqf** [comment]
will be implemented via Spawn() effect over entity. Yeah, thats it.

**Comment by fqqf** [comment]
emote_start
emote_nervous
emote_lenny
emote_mark
emote_end

**Comment by fqqf** [comment]
image

**Comment by fqqf** [comment]
https://github.com/user-attachments/assets/a9ea240d-8c20-4db1-bc8b-a0d8c6a005db

File: Resources/Prototypes/_StarLight/Entities/Effects/cloud_emotes.yml
Comment (Rinary1) [inline Resources/Prototypes/_StarLight/Entities/Effects/cloud_emotes.yml:1]: parents? Like, basic emotion with default things, like TImedDespawn, + you need:
```
- type: Transform
  noRot: true
  ```

**Review by Rinary1 — CHANGES_REQUESTED** [review]

**Comment by fqqf** [comment]
ready to review


https://github.com/user-attachments/assets/fb5a70e3-0e67-48e2-93a6-37d0b64c72a8

**Comment by fqqf** [comment]
not my linter, its from cluwne

**Review by Rinary1 — APPROVED** [review]

**Comment by fqqf** [comment]
oops, forgot changelog. Fixed

**Comment by CrimeMoot** [comment]
it's so cute

**Pull request merged**